docs(readme): add AwesomeTechStack to lighthouse integrations (#10475)
GoogleChrome/lighthouse: readme.md

* **[Web Page Test](https://www.webpagetest.org)** — An [open source](https://github.com/WPO-Foundation/webpagetest) tool for measuring and analyzing the performance of web pages on real devices. Users can choose to produce a Lighthouse report alongside the analysis of WebPageTest results.

> * **[AwesomeTechStack](https://awesometechstack.com)** — AwesomeTechStack is a free to use website tech stack analyzer. AwesomeTechStack analyzes and rates the performance of technology stacks. AwesomeTechStack gives you insights about state-of-the-arts, up-to-dateness and secureness of any website's technology stack, with guidance to improve performance. Lighthouse insights are a crucial part of a website's tech stack rating.

Reviewer: this is an alphabetized list, so you can actually put this first :)

Reviewer: ```suggestion
* **[AwesomeTechStack](https://awesometechstack.com)** — AwesomeTechStack is a free to use website tech stack analyzer. AwesomeTechStack provides insights into the security, modernity, and performance of any website's technology stack and guidance to improve performance. Lighthouse insights are a crucial part of a website's tech stack rating.
```

some of the phrasing felt a bit unusual/repetitive to me personally, so some optional suggestions for improvements if you're up for them :)

Author: Hello @patrickhulce 

Wow! Thank you for your detailed report and suggestions.
Yes. I am the main maintainer of the project.

We will start to work in, your suggestions immediately. And thank you for finding the small things developers are mostly blind for when they are working on it -> 0-39 :)

We will rephrase some texts, as this is already a ticket on our Roadmap, we will do some necessary adaptions now.

You wrote:
>  "It'd be awesome to be able to actually open up the Lighthouse report that underlies the quality score"

Yes. There was a mistake in our documentation and "lack of transparency" I would say.
The Quality score comes from Best Practices, Accessibility and PWA.

Performance and SEO are characteristics for a good tech stack too, but the main fact that performance is related to the network makes it not suitable for our Algorythm.

Yes, that was one thing we were discussing about this week. First we wanted to add this functionality for users which have a login (which is not yet live). To store the reports in our database needs quite some space :). 
But we will show the lighthouse report on demand when clicking on "view report" which will be added soon, then we will send the report as html and render it on our website.

The awesomeness score is a combination of quality, powered by lighthouse and technology stats gathered all over open source community tools, like Github stars, Maintaners, pulse, last release, vulnerabilities, Npm downloads, jsdeliver stats etc..

I will update the pull request when our changes are live.
